api/src/services/dwc-service.ts

```diff
@@ -90,4 +70,60 @@ export class DwCService extends DBService {
     // Apply patch operations
     return jsonpatch.applyPatch(jsonObject, patchOperations).newDocument;
   }
+
+  async decorateDWCASourceData(occurrenceSubmissionId: number): Promise<boolean> {
```

Comment:
We should add some tests to this too, make sure there are no surprise circumstances that break this.